### DISCUSSION TITLE
Dual tool?

### DISCUSSION THREAD

**Discussion by kloel** [AxxAxx/AxxSolder#51] [Ideas]
Is this design capable of handeling dual tool use?

ofcourse one tool wil be used and one wil rest at a stand, but when one tool is not sitting right in his stand and the station wanne power 2 tools?

**Comment by AxxAxx** [comment]
No, at the moment it only supports one tool. 

Possible that it could be done in the future. Maybe then also a tweezer. But not today.

**Reply by AxxAxx** [reply]
And that would require new hardware

**Comment by AxxAxx** [comment]
Look at this! https://github.com/AxxAxx/AxxSolder/discussions/151
